com.unity.ml-agents/CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ and this project adheres to
   C# style conventions. All public fields and properties now use "PascalCase"
   instead of "camelCase"; for example, `Agent.maxStep` was renamed to
   `Agent.MaxStep`. For a full list of changes, see the pull request. (#3828)
+- Update Barracuda to 0.7.0-preview.
```

Comment:
note: we already have `Updated Barracuda to 0.6.3-preview` below - we should combine that with this.